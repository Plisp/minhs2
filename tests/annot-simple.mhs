main =
    let id :: Int -> Int x = x;
    in id;
