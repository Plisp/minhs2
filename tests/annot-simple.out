main :: (Int -> Int) = let id :: (Int -> Int) x = x;
                       in id;
